.github/workflows/report-acceptance-criteria.yml

```diff
@@ -0,0 +1,30 @@
+name: Report Acceptance Criteria
+run-name: Generate AC coverage report
+
+# The report generation is performed ad hoc via manual invocation.
+on: workflow_dispatch
+
+env:
+  glossaryURL: https://github.com/ameteiko/goreleaser/wiki/Glossary.md
+  # The report highlighter utility that is checked out from the
+  # https://github.tools.sap/pandas/coverage-metrics/blob/main/bin/utils/report-highlighter/highlighter.py
+  highlighter: ${{ github.workspace }}/scripts/bin/utils/report-highlighter/highlighter.py
+
+jobs:
+  generate-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out LM source code
+        uses: actions/checkout@v4
+      - name: Check out report highlighter
+        uses: actions/checkout@v4
+        with:
+          github-server-url: https://github.tools.sap
```

Comment:
I don't think it's a good idea cross reference script from another repo, especially from a sap internal one, please relocate this script to LM.